noir-projects/noir-contracts/contracts/app/token_blacklist_contract/src/main.nr

```diff
@@ -46,8 +46,8 @@ pub contract TokenBlacklist {
         address::AztecAddress, constants::MAX_NOTE_HASHES_PER_TX, traits::Packable,
     };
 
-    // Changing an address' roles has a certain delay before it goes into effect. Here set to 2 slots.
-    global CHANGE_ROLES_DELAY: u64 = 72;
+    // Changing an address' roles has a certain delay before it goes into effect. Here set to 1 day.
+    global CHANGE_ROLES_DELAY: u64 = 86400;
```

Comment:
With better cheatcodes we can put here a reasonable delay instead of the original 2 slots (SharedMutable should never be used with such shorts delays).